src/core/qgsofflineediting.cpp

```diff
@@ -817,7 +831,7 @@ void QgsOfflineEditing::applyAttributeValueChanges( QgsVectorLayer* offlineLayer
   for ( int i = 0; i < values.size(); i++ )
   {
     QgsFeatureId fid = remoteFid( db, layerId, values.at( i ).fid );
-
+    QgsDebugMsg( QString( "Offline changeAttributeValue %1 = %2" ).arg( QString( attrLookup[ values.at( i ).attr ] ), values.at( i ).value ) );
```

Comment:
Is there something left to fix?
This call here is probably overly verbose in a real life scenario.
And some others could probably directly go to the message log and make the user aware of a potential problem (when there are no debug builds)